src/bin/stratis-utils.rs

```diff
@@ -111,6 +111,54 @@ fn predict_usage(
             ))
         })?;
 
+    // Guess at size required by specified filesystems. Due to layout effects
+    // we can not predict precisely how much space will be required by xfs
+    // metadata, etc. We use the following formulas which are based on some
+    // observations of filesystem sizes on Stratis pools.
+    // If logical size (ls) requested is:
+    // <= 4 TiB, expected size use is 10 ^ (0.51 * log_10(ls) + 3)
+    // otherwise, expected size is 10 ^ (0.36 * log_10(ls) + 5)
+    // Sizes less than 512 MiB are rejected.
+    // For our data, this computation always overestimates the amount of the
+    // data required by the a filesystem, but always by less than 100%.
```

Comment:
Should this be "by the filesystem" or "by a filesystem"?

Reply:
Fixed in subsequent commit.